connectors/src/connectors/intercom/lib/intercom_api.ts

```
@@ -90,12 +91,21 @@ async function queryIntercomAPI({
 
     return response;
   } catch (e) {
-    logger.info(
-      { path, response: text, status: rawResponse.status },
-      "Failed to parse Intercom JSON response."
-    );
-
-    throw e;
+    if (rawResponse.status === 405) {
+      const isCaptchaError = text.includes("captcha-container");
+      const workflowError: WorkflowError = {
```

Can we throw a proper Error class?

---

As per IRL I created a card!